liquigraph-core/src/test/java/org/liquigraph/core/io/ChangelogGraphReaderTest.java

```diff
@@ -92,7 +85,34 @@ public void reads_changeset_with_multiple_queries() throws SQLException {
         }
     }
 
+    @Test
+    public void reads_changeset_with_multiple_queries_in_order() throws SQLException {
```

Comment:
I wanted to leave the previous test like it was, but I wanted to also test that the reader returned the list of queries in the order of execution. I will refactor a bit and also revert my formatting changes (auto-formatting was on and I use a modified google code formatter).